web/cobrands/sass/_base.scss

```diff
@@ -2823,6 +2823,13 @@ $nicetable-hover-background: rgba($primary, 0.15) !default;
     }
 }
 
+.emergency-message {
+  border: 1px solid #525252;
```

Comment:
Might be guilding the lilly here, but should we perhaps make a variable for this colour, in `_colours.scss`, or even just as a `!default` right before this ruleset, so it can be customised more easily for new cobrands? eg:

```scss
$emergency-message-border: 1px solid #525252;

.emergency-message {
  border: $emergency-message-border;
  …
}